src/vcpkg/vcpkgpaths.cpp

```diff
@@ -49,7 +49,7 @@ namespace
         Filesystem& filesystem, const Path& root, std::string* option, StringLiteral name, LineInfo li)
     {
         auto result = process_input_directory_impl(filesystem, root, option, name, li);
-        Debug::print("Using ", name, "-root: ", result, '\n');
+        msg::write_unlocalized_text_to_stdout(Color::none, fmt::format("Using {}-root: {}\n", name, result));
```

Comment:
I think PR build is failing because this was Debug::print but you upgraded it to always print.